tools/iso/preseed.cfg

```diff
@@ -0,0 +1,1413 @@
+# No partitions to encrypt
+# for internal use; can be preseeded
+d-i	preseed/run/checksum	string	
+# Go back to the menu and correct this problem?
+# Write previous changes to disk and continue?
+# Device file for accessing the CD-ROM:
+d-i	cdrom-detect/cdrom_device	string	/dev/sdb1
+# Network autoconfiguration failed
+netcfg	netcfg/dhcp_failed	note	
+# No boot loader installed
+nobootloader	nobootloader/confirmation_common	note	
+# LILO installation target:
+lilo-installer	lilo-installer/manual_bootdev	string	
+# Enable source repositories in APT?
+apt-setup-udeb	apt-setup/enable-source-repositories	boolean	true
+# Select your time zone:
+# Choices: Eastern, Central, Mountain, Pacific, Alaska, Hawaii, Arizona, East Indiana, Samoa
+tzsetup-udeb	time/zone	select	US/Eastern
```

Comment:
I'm not sure if I'm reading this file right, but could we set the timezone to UTC by default?

Reply:
Fixed by c23c9beb